### ISSUE TITLE
[Keyboard Navigation - API Management Iframe - Other Policies] ‘Expand(-) and Collapse(+) buttons are not accessible using keyboard present inside Policies editor.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1800]
**User Experience:**
User rely on keyboard/Screen Reader will find difficulty to interact with the controls if controls are not accessible through keyboard as user won't be able to perform functionality over such control.
 
Note: User credentials should NOT be included in the bug.
 
**Repro Steps:**
1. Open Azure portal with valid credentials.
2. Navigate to the any listed "API Management services" and select it.
3. Navigate to the "API" Option and select it.
4. Navigate to the "All API" button and select it.
5. Navigate to the "Design" pane and select Add Policies button.
6. Navigate to the "Add Inbound Policies" blade and select "Other Policies" tile.
7. Navigate to the various control of the "Other Policies" blade.
8. Verify whether all the controls accessible via keyboard inside Policies editor.
9. Verify the issue.
 
**Actual Result:**
‘Expand(-) and Collapse(+) buttons are not accessible using keyboard present inside Policies editor.
 
**Expected Result:**
‘Expand(-) and Collapse(+) buttons should be accessible using keyboard present inside Policies editor.

**Note:**
Similar issue exist with Chrome.
Similar issue exist throughout the application

**Test Environment:**
Version: 2004 (OS Build: 19041.1)
Browser: Edge, Chrome
#URL:https://ms.portal.azure.com

 
**MAS Reference:**
MAS 2.1.1 - Keyboard (36)

**Reference Links**
Accessibility Insights! - Identify accessibility bugs before check-in and make     bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
  
 Attachment : 
[MAS2.1.1_expand button.zip](https://github.com/microsoft/monaco-editor/files/4146925/MAS2.1.1_expand.button.zip)

Bug copied from: API management Iframe: https://msazure.visualstudio.com/One/_workitems/edit/6102153

**Comment by alexdima** [comment]
I have watched the video. The `+` and `-` refer to the folding collapse/expand icons. Let's track in https://github.com/microsoft/vscode/issues/43262
